cfg/config.go

```diff
@@ -55,8 +55,8 @@ type Config struct {
 	Ardop     ArdopConfig     `json:"ardop"`      // See ArdopConfig.
 	Telnet    TelnetConfig    `json:"telnet"`     // See TelnetConfig.
 
-	// Address and port to a GPSd daemon for position reporting.
-	GPSdAddr string `json:"gpsd_addr"`
```

Comment:
You are absolutely right about the reliability of the GPS time. However, as there are still older GPS devices in use of which some unfortunately have encounterd week roll-over issues like the one we have on board (Furuno SC-60 [Issue](https://www.furunousaforum.com/viewtopic.php?f=12&t=4073)). For this reason I was forced to get another source for the current time. 

We could remove the struct and make just a second variable in order not to break compatibility:
```go
GPSdAddr:                  "localhost:2947", // Default listen address for GPSd
GPSdUseServerTime: false,                    // Use server time instead of GPS time
```

Reply:
Ah, I see your point.

I believe we can write some code that parses the legacy `gpsd_addr`. I'll fix that once the feature is merged. Just keep (new) structure as initialy proposed in this PR.